user_pinger.py

```diff
@@ -582,6 +604,13 @@ def create_group(body: str, author: praw.models.Redditor) -> None:
             Note:
             Moderator will be a member of the group
             """
+            group_name: str = body.upper()
+            ret: bool, msg: Optional[str] = self._validate_group_name(group_name)
```

Comment:
Same issue here as above

```suggestion
            ret: bool
            msg: Optional[str]
            ret, msg = self._validate_group_name(group_name)
```